#!/usr/bin/python3
# A function that raises name exception with message

def raise_exception_msg(message=""):
    rasieNmaeError(message)
